default/scripting/species/common/troops.macros

```diff
@@ -23,6 +23,16 @@ BASIC_DEFENSE_TROOPS
             effects = SetMaxTroops value = Value + 4
             accountinglabel = "HOMEWORLD_LABEL"
 
+        EffectsGroup
+            scope = Source
+            activation = And [
+                Planet
+                Not Unowned
+            ]
+            priority = [[VERY_EARLY_PRIORITY]]
+            effects = SetMaxTroops value = Value + 6
```

Comment:
the six is a magic number, better define it via a macro or a named valueref

Reply:
I missed this fix in my last force-push. TODO.

Reply:
Defined the value in named_values.focs.txt.